resources/report-table-formatter.js

```diff
@@ -41,8 +41,8 @@
 
     for (var v of variants) {
         var split = v.split(':');
-        vafs[split[0]].push(split[2]);
-        vats[split[0]].push(split[1]);
+        vafs[split[0]].push(split[2].trim());
```

Comment:
This leads to an error in the generated report of the CI somehow: 
` undefined is not an object (evaluating 'split[2].trim')`
I am not sure why the `trim()` should cause that though.